sections/battlefield.tex

```diff
@@ -0,0 +1,443 @@
+        \lightcell[1.8]{Draw 4 Ability cards and keep 3.} &
+        \lightcell[1.8]{Draw 6 Ability cards and keep 4.} \\
+        \darkcell[1.8]{Artifact cards} &
+        \lightcell[1.8]{Draw 2 Artifact cards and keep 1.} &
+        \lightcell[1.8]{Draw 3 Artifact cards and keep 2.} &
+        \lightcell[1.8]{Draw 4 Artifact cards and keep 3.} \\
+        \darkcell[2.2]{Spell cards} &
+        \lightcell[2.2]{Draw 1 Spell card for~a~\svg[12]{might-yellow} Hero or 2 Spell cards for a \svg{magic-yellow} Hero.} &
+        \lightcell[2.2]{Draw 2 Spell cards for a \svg[12]{might-yellow} Hero or 4 Spell cards for a \svg{magic-yellow} Hero.} &
+        \lightcell[2.2]{Draw 3 Spell cards for a \svg[12]{might-yellow} Hero or 5 Spell cards for a \svg{magic-yellow} Hero.} \\
+        \darkcell[2.2]{Units} &
+        \lightcell[2.2]{Use all \svg{bronze} and \svg{silver} units on the ``Few'' side.} &
+        \lightcell[2.2]{Use all \svg{bronze} and \svg{silver} units on the ``Pack'' side.} &
+        \lightcell[2.2]{Use all units (\svg{bronze}, \svg{silver}, and \svg{golden}) on the ``Pack'' side.} \\
+      \end{tabularx}
+    }
+  \begin{center}
+    \vspace*{1em}
+    \transparent{0.2}\includegraphics[width=0.35\linewidth]{\art/corpse.png}
+  \end{center}
+\end{expansion}
+
+\begin{expansion}[before=\vspace*{-11mm}]{battlefield}
+  \pagetarget{Battlefield Combat}{\subheader{Battlefield Combat}}
+  \begin{multicols*}{2}
+  Playing on the Battlefield Board requires using Miniatures for Combat instead of unit cards.
+
+  \subsection*{\pagetarget{BF Obstacles}{Obstacle Tokens}}
+  You can place three token types on the Battlefield Board: Effect, Obstacle and Wall/\allowbreak Gate.
+  Unless otherwise stated, these Tokens count and work as \pagelink{Combatterminology}{Combat Obstacles}.
+  To place Effect Obstacles on the Battlefield Board you'll need the card which creates the effect, such as the Fire Wall Spell: use the Tokens and put the card beside the Battlefield as a reminder.
+  Effect Obstacles may be entered by units, if the card text allows it.
+  \begin{figure}[H]
+    \centering
+    \begin{subfigure}[b]{0.3\linewidth}
+      \centering
+      \includegraphics[width=\linewidth]{\images/bf-obstacle-1.png}
+      \caption{\imagecaption{Effect}}
+    \end{subfigure}
+    \begin{subfigure}[b]{0.3\linewidth}
+      \centering
+      \includegraphics[width=0.9\linewidth]{\images/bf-obstacle-2.png}
+      \caption{\imagecaption{Obstacle}}
+    \end{subfigure}
+    \begin{subfigure}[b]{0.3\linewidth}
+      \centering
+      \includegraphics[width=0.9\linewidth]{\images/bf-obstacle-3.png}
+      \caption{\imagecaption{Wall/Gate}}
+    \end{subfigure}
+  \end{figure}
+  \vspace*{-0.5\baselineskip}
+  \note{3}{
+    All unit Miniatures also count as Obstacles.
+  }
+
+  \vspace*{-1em}
+  \subsection*{Combat Setup}
+  \setlength\intextsep{0pt}
+  \setlength\columnsep{1em}
+  \begin{wrapfigure}{r}{0.33\linewidth}
+    \centering
+    \footnotesize
```

Comment:
```suggestion
```

imagecaption already includes this

Reply:
The two line "Initiative Token" still looks badly centered like in the other MR. How can we fix this?